gcc/testsuite/rust.test/xfail_compile/unsafe.rs

```diff
@@ -1,4 +1,5 @@
 fn main() { // { dg-ice "#382" }
     unsafe {
     }
+    ()
```

Comment:
I interpret the purpose of this test as testing whether or not `dg-ice` works well. Without a trailing `()`, the unsafe block is a trailing expression, therefore there is a fatal error instead of an internal compiler error because of the following code segment:
https://github.com/Rust-GCC/gccrs/blob/dd3c58b2ce55ce6585916607927c3bca087610c6/gcc/rust/hir/rust-ast-lower-expr.h#L60-L65